examples/README.md

```diff
@@ -0,0 +1,85 @@
+# krun example
+
+This directory contains a simple experiment using krun.
+This is a good starting point for setting up your own krun configuration.
+
+The example here contains two benchmark programs (*nbody* and *dummy*),
+executed on two VMs (*cPython* and a standard *JVM* such as HotSpot).
+Each benchmark is run for 5 iterations on the same VM, then the VM is
+restarted and the benchmark is re-run for another 5 iterations.
+We say that the benchmarks are *executed* twice, with 5 *iterations*.
```

Comment:
Maybe use a different number of iterations and executions, so as to help distinguish them?